app/src/main/res/values/dimens.xml

```diff
@@ -7,4 +7,7 @@
   <dimen name="audio_fragment_margin">32dp</dimen>
   <dimen name="cellular_data_dialog_padding">24dp</dimen>
   <dimen name="state_previous_next_button_radius">4dp</dimen>
+
+  <dimen name="padding_checkbox">18dp</dimen>
```

Comment:
I would suggest renaming this to `train_checkbox_padding` because checkboxes are in ItemSelectionInput too and this can become confusing in future.

Reply:
Done.